helpdesk_mgmt_project/models/helpdesk_ticket.py

```diff
@@ -14,6 +16,34 @@ class HelpdeskTicket(models.Model):
         store=True,
     )
 
+    @api.model
+    def _default_company_id(self):
+        if self._context.get("default_project_id"):
```

Comment:
I think that this is not safe enough.
I've already have a case where a variable in context was setup with None value.
So it was present in the context but not usefull for browsing and throw an execpetion.
Please adapt

Reply:
If the variable has a None value, `self._context.get("default_project_id")` will evaluate to False, so should not pose a problem.

Reply:
Hi @NL66278 , 
Thanks, sadly it's not what I'm experiencing  : 
![image](https://github.com/user-attachments/assets/f855dc96-34f1-4f2f-885f-9016a6298847)

Reply:
That is because None is not an instance of bool. But that is not how the if statement evaluates False like values. In `if x:` the condition will not be satisfied if x is None, False, 0, an empty list, an empty dictionary, an empty string....